<launch>

 <node pkg="hand_camera_manip" type="camera_manip_node" name="camera_manip_node" output="screen">
    <param name="kp_arm_lift" value="0.005"/>
    <param name="kp_arm_flex" value="0.08"/>
    <param name="kp_arm_roll" value="0.5"/>
    <param name="kp_wrist_flex" value="0.5"/>
    <param name="kp_wrist_roll" value="0.01"/>
    <param name="y_center" value="180"/>
    <param name="x_center" value="120"/>
    <param name="area_setpoint" value="0"/>
    <param name="kp_base_x" value="0.1"/>
    <param name="kp_base_y" value="0.1"/>
    <param name="routine" value="1"/>
   </node>
</launch>
